### DISCUSSION TITLE
Multi stepper forecasted path monitoring

### DISCUSSION THREAD

**Discussion by meshein** [gin66/FastAccelStepper#274] [Q&A] [answered]
I have a unique system with two AC servo motors driving independent objects on the same track. This works most of the time but there is an edge case where when in close proximity a change of direction can cause a clash.  An example is to view object 1 traveling to the right towards object 2 which is currently stopped.  If both are given a command to move to the left, object 2 will immediately start moving while object 1 must decelerate first, therefore the target paths collide.

I plan to add some logic to hold the stationary object until the moving one has come to rest first.  But is there a better way to forecast the paths?

Side note, I’m using linear acceleration but I must be misinterpreting the linear acceleration formula in the docs because it seems to apply differently. 

Thank you!

**Comment by gin66** [comment, marked as answer]
please check the latest version on github, which provides `stepsToStop()`. Perhaps this helps.
